### PR TITLE
feat(components): icon & progress-ring full int sizing spectrum

### PR DESCRIPTION
**Pull request by yinonov**
getting ready to vivid-3 stable

### CODE REVIEW

File: libs/components/src/lib/icon/README.md [hunk 1]
```diff
@@ -48,15 +48,24 @@ While `type` fetches an icon from the Vivid predefined set of icons, `svg` prope
 
 ### Size
 
-Use the `size` attribute to apply a predefined size to the icon.
+Use the `size` attribute/property to set the icon's size.
 
-- Type: `'condensed'` | `'normal'` | `'expanded'`
-- Default: `'normal'`
+- Type: `-6` | `-5` | `-4` | `-3` | `-2` | `-1` | `0` | `1` | `2` | `3` | `4` | `5`
+- Default: `0`
```
Comment (rachelbt) [inline libs/components/src/lib/icon/README.md:54]: 0 is not the default. the default is the font size of the parent....

File: libs/components/src/lib/progress-ring/README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Use the `paused` attribute to indicate the progress is paused.
 
 Use the `size` attribute/property to set the progress ring's size.
 
-- Type: `-5` | `-4` | `-3` | `-2` | `-1` | `0` | `1` | `2` | `3`
+- Type: `-5` | `-4` | `-3` | `-2` | `-1` | `0` | `1` | `2` | `3` | `4` | `5`
 - Default: `0`
```
Comment (yinonov) [inline libs/components/src/lib/progress-ring/README.md:74]:
```suggestion
- Default: `1em`
```

File: libs/components/src/lib/progress-ring/README.md [hunk 3]
```diff
@@ -70,8 +70,8 @@ Use the `paused` attribute to indicate the progress is paused.
 
 Use the `size` attribute/property to set the progress ring's size.
 
-- Type: `-5` | `-4` | `-3` | `-2` | `-1` | `0` | `1` | `2` | `3`
-- Default: `0`
+- Type: `-5` | `-4` | `-3` | `-2` | `-1` | `0` | `1` | `2` | `3` | `4` | `5`
+- Default: _inherited font-size_
```
Comment (yinonov) [inline libs/components/src/lib/progress-ring/README.md:74]:
```suggestion
- Default: `0`
```

File: libs/components/src/lib/icon/README.md (hunk 1 above)
Comment (yinonov) [reply libs/components/src/lib/icon/README.md:54]:
```suggestion
- Default: `'1em'` (_inherited font-size_)
```

File: libs/components/src/lib/icon/README.md [hunk 4]
```diff
@@ -48,15 +48,24 @@ While `type` fetches an icon from the Vivid predefined set of icons, `svg` prope
 
 ### Size
 
-Use the `size` attribute to apply a predefined size to the icon.
+Use the `size` attribute/property to set the icon's size.
 
-- Type: `'condensed'` | `'normal'` | `'expanded'`
-- Default: `'normal'`
+- Type: `-6` | `-5` | `-4` | `-3` | `-2` | `-1` | `0` | `1` | `2` | `3` | `4` | `5`
+- Default: `'1em'` (_inherited font-size_)
```
Comment (rachelbt) [inline libs/components/src/lib/icon/README.md:54]:
```suggestion
- Default: `'1em'` (_inherit font-size_)
```

**Review by rachelbt — APPROVED** [review]
LGTM

**Pull request merged**